### PR TITLE
Update CG-12-20.md

### PR DESCRIPTION
**Pull request by KloudKoder**

### CODE REVIEW

**Comment by dschuff** [comment]
This seems fine to me; there is a risk that because it's the week before Christmas that we won't have as many participants as we usually do. (I'll be available to run the meeting though). I wouldn't want to schedule a vote or something like that but it might be ok if you just want to give an update, have discussion, etc.

**Comment by dschuff** [comment]
Ah, I see, it sounds like what you want is to ask for a [phase 1](https://github.com/WebAssembly/meetings/blob/main/process/phases.md) proposal. Given how the discussion on the issue has gone, this seems appropriate to me too (the bar for phase 1 isn't too high).

Deepti/Conrad, what do you think of just scheduling a phase 1 vote here? Do we have some minimum level of quorum below which we'd be uncomfortable letting something advance to phase 1? (I have less concern about that than I would for later phases I think).

**Comment by conrad-watt** [comment]
Personally I think it's fine for the vote to go ahead although I won't be able to attend the meeting. I agree that we probably can't be so lax for any phase >1 proposal though.

Given the timing, I think it would also be a good idea to give a separate update presentation next year if/when it looks like a phase 2 vote is on the horizon, just so the presentation for that vote isn't the first time some people are seeing the proposal.

File: main/2022/CG-12-20.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ Installation is required, see the calendar invite.
     1. Introduction of attendees
 1. Find volunteers for note taking (acting chair to volunteer)
 1. Proposals and discussions
-1. Closure
+    1. Opcode proposal for new instructions for interval arithmetic (Kloud Koder)[20 mins]
```
Comment (conrad-watt) [inline main/2022/CG-12-20.md:26]:
```suggestion
    1. Phase 1 poll: [alternate floating-point rounding mode instructions](https://github.com/WebAssembly/design/issues/1456) (Kloud Koder)[20 mins]
```

**Comment by dtig** [comment]
I think a vote for Phase 1 seems like a good next step, and the scheduling seems fine, especially as we've talked about a rounding mode control in the CG before, and there's been some discussion in the linked issue and the [previous issue](https://github.com/WebAssembly/design/issues/1384) that the prototyping issue pulls from.

**Review by dschuff — APPROVED** [review]

**Pull request merged**